org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/customservice/synapse/expression/ExpressionCompletionsProvider.java

```diff
@@ -128,7 +128,9 @@ private static MediatorTryoutInfo getMediatorProperties(ICompletionRequest reque
         }
         String projectPath = getProjectPath(request.getXMLDocument().getDocumentURI());
         ServerLessTryoutHandler serverLessTryoutHandler = new ServerLessTryoutHandler(projectPath);
-        String payload = ExpressionCompletionUtils.getInputPayload(projectPath);
+
+        String documentUri = request.getXMLDocument().getDocumentURI().substring(7);
```

Comment:
Shall we add a comment here for future reference on why we use the value 7?

Reply:
fixed in https://github.com/wso2/mi-language-server/pull/397/commits/75a54df49de615e2235b14fef17e2c82a4bddcd6